shim/test/realm/test-repair.js

```diff
@@ -25,7 +25,7 @@ function testForBug(o) {
 }
 
 test('fix the bug in which accessor methods leak the global', t => {
-  const unfixedGlobal = getNewUnsafeGlobal();
+  const unfixedGlobal = vm.runInNewContext('this');
```

Comment:
The old code was host independent. This seems better than the new code.

Reply:
True. However, the old code used the shim to create an artifact (unfixedGlobal) for testing without testing it, and increased code coverage artifically. 

I have not added browser testing yet, however I will fix this in preparation.